lib/modules/manager/gitlabci-include/common.ts

```diff
@@ -0,0 +1,35 @@
+import is from '@sindresorhus/is';
+import type {
+  GitlabInclude,
+  GitlabIncludeLocal,
+  GitlabIncludeProject,
+  GitlabPipeline,
+} from '../gitlabci/types';
+
+export function isGitlabPipeline(obj: any): obj is GitlabPipeline {
+  return is.object(obj) && Object.keys(obj).length !== 0;
+}
+
+export function filterIncludeFromGitlabPipeline(
+  pipeline: GitlabPipeline
+): GitlabPipeline {
+  const pipeline_without_include = {};
+  for (const key of Object.keys(pipeline).filter((key) => key !== 'include')) {
+    Object.assign(pipeline_without_include, {
+      [key]: pipeline[key as keyof typeof pipeline],
+    });
```

Comment:
We use Object.assign() very sparingly due to its potential to cause vulnerabilities. Can you find an alternative approach?